packages/xod-project/test/flatten.spec.js

```diff
@@ -1472,7 +1474,9 @@ describe('Flatten', () => {
         expect(flatProject.isRight).to.be.true();
         Helper.expectEither(
           (project) => {
-            expect(project).to.deep.equal(castMultipleOutputsFlat);
+            expect(project).to.deep.equal(
+              Helper.defaultizeProject(castMultipleOutputsFlat)
+            );
```

Comment:
Yeah, I know, this code is pretty old, when we were not so savvy in FP. But now, isn’t it more clear to write:

```js
foldEither(
  err => assert(false, `Expected Either.Right, got Either.Left: ${err}`),
  project => assert.deepEqual(project, Helper.defaultizeProject(castMultipleOutputsFlat),
  flatProject
);
```

Less lines, less nesting.

Also, I think it would be better to read the xodballs with `fs` module and use “defaultize” right away. This way the fixtures would be manageable by XOD IDE out of the box and the test will be more self-contained.